.github/workflows/ci.yml

```diff
@@ -11,28 +18,41 @@ jobs:
         uses: actions/checkout@v3
 
       - name: 🎛 Setup Node
+        if: github.event.action != 'closed'
         uses: actions/setup-node@v3
         with:
           node-version: 16.16
-          cache: 'npm'
+          cache: 'yarn'
 
       - name: ⚙️ Install
-        run: npm ci
+        if: github.event.action != 'closed'
+        run: yarn
 
       - name: 🎨 Lint JS
-        run: npm run lint:js
+        if: github.event.action != 'closed'
+        run: yarn lint:js
 
       - name: 🎨 Lint CSS
-        run: npm run lint:css
+        if: github.event.action != 'closed'
+        run: yarn lint:css
 
       - name: 🎨 Lint Format
-        run: npm run lint:format
+        if: github.event.action != 'closed'
+        run: yarn lint:format
 
       - name: 🎨 Lint Types
-        run: npm run lint:types
-
-      - name: 🛠 Build
-        run: npm run build
+        if: github.event.action != 'closed'
```

Comment:
The normal deploy job takes over when the PR is closed so we don't really need to build a preview for a closed item since the preview action will just remove the PR build from gh pages.